apps/web/src/pages/Applications/ApplicationProfilePage/utils.ts

```diff
@@ -0,0 +1,112 @@
+import { MessageDescriptor, defineMessage } from "react-intl";
+import {
+  PencilSquareIcon,
+  ExclamationCircleIcon,
+  CheckCircleIcon,
+  InformationCircleIcon,
+} from "@heroicons/react/24/outline";
+import { HeadingProps } from "@gc-digital-talent/ui";
+import { commonMessages } from "@gc-digital-talent/i18n";
+import { SectionKey } from "./types";
+
+type IconType = React.ForwardRefExoticComponent<React.SVGProps<SVGSVGElement>>;
+
+interface GetSectionIconArgs {
+  isEditing: boolean;
+  fallback: IconType;
+  completed?: boolean | null;
+  error?: boolean | null;
+}
+
+export type SectionIcon = {
+  icon: IconType;
+  color?: HeadingProps["color"];
+};
+
+type GetSectionIconFn = (args: GetSectionIconArgs) => SectionIcon;
+
+export const getSectionIcon: GetSectionIconFn = ({
```

Comment:
Is this working?  I'm seeing only editing and fallback icons on the page.

Reply:
lol, nope it definitley is not working since I forgot to update the args to no longer be hard coded 🤦‍♀️ 

Though, this has me wondering. How should we determine "completed" status for the optional equity section. Since it is optional it will always be in completed status. Though, maybe that is not an issue 🤔

Reply:
Yeah, I imagine DEI would never be in error.

Reply:
Okay, it is working now after [fix section icon logic](https://github.com/GCTC-NTGC/gc-digital-talent/pull/6296/commits/6ac41504564884a24cc676f0acf0e5b85985771d)

Reply:
Looks perfect.  Glad those validators are coming in useful.  :smile: